packages/itwinui-css/src/tile/tile.scss

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import '../button/index';
```

Comment:
Idk if this is known but icons background is missing 
![image](https://user-images.githubusercontent.com/36186912/185617135-5a1492c8-ada4-4cb4-874e-7bd0f6d63094.png)

Reply:
where is that from? i'm not seeing any missing icons

Reply:
ok, why background was changed from darker to this, barely visible one? Now it is harder to see that there is an icon in the corner if image is bright and previously we didn't have such problem since icon background was darker.

Reply:
I'm still not sure what part you're talking about. 

https://itwin.github.io/iTwinUI/718/tile.html and https://itwin.github.io/iTwinUI/tile.html look the same

Reply:
I am seeing this in local build. As usual build must be fucked up...